service.subtitles.napisy24pl/service.py

```diff
@@ -306,6 +89,7 @@ def get_params():
 
     return param
 
+
 params = get_params()
```

Comment:
This is how amet recommended to write subtitle addon, look at the source code of opensubtitle that he wrote:
https://github.com/xbmc/repo-scripts/blob/gotham/service.subtitles.opensubtitles/service.py#L128